hardware/panels.py

```diff
@@ -43,6 +44,20 @@ def dot(self, sun_xyz):
         buffer = self.area*np.dot(sun_xyz, self.normal_rot)
         buffer[buffer<0] = 0.0
         return buffer
+
+    ### ---
+    def pv_angle_corr(self, sun_xyz):
+        #Correction factor for measured divergence of PV power as a function of solar angle from expected cosine/dot-product dependence 
```

Comment:
Add a note to self here that will allow you to come back to this in 10 years. I.e. something like "See Ben Sawlianchik's private blablah.ipynb for how to the fitting function was determined."

Reply:
Will add in next commit. I will also add the "apply_cos_correction" to the devices.yml file and the __init__ function for the panel class.

Reply:
Yes, keep the PR open, I should see your update commits.